frontend/src/store/queries/cars/index.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  BrandType,
+  ModelType,
+  OptionsType,
+} from '@autoline/shared/common/types/types';
+import { API } from '@store/queries/api_routes';
+
+import { api } from '..';
+
+export const carsApi = api.injectEndpoints({
+  endpoints: (builder) => ({
+    getBrands: builder.query<BrandType[], void>({
+      query: () => `${API.CARS}/brands`,
+    }),
+    getModelsOfBrand: builder.query<ModelType[], string>({
+      query: (brandId) => `${API.CARS}/brand/${brandId}/models`,
+    }),
+    getUsedOptions: builder.query<OptionsType, void>({
+      query: () => `${API.CARS}/options`,
+    }),
+    getFilteredCars: builder.query<void, string[][] | undefined>({
+      query: (params) => {
+        return {
+          url: `${API.CARS}/search`,
+          params: params,
+        };
```

Comment:
Use braceless one-liner syntax here

Reply:
It doesn't work when i use this line `query: (params) => ${API.CARS}/search/${params}`
Probably because I don't pass an object in params, but rather an array of arrays containing params.
![image](https://user-images.githubusercontent.com/49813216/186493564-c285e75d-2aa4-4758-87e4-d0b9f1d69a62.png)

Reply:
It shouldn't work, because in `query: (params) => ${API.CARS}/search/${params}` you're trying to pass the **_object_** to as the part of the _**string**_.

The one-liner syntax I mentioned is:
```suggestion
      query: (params) => ({
          url: `${API.CARS}/search`,
          params,
      });
```